Manual inject expectation doc issue/101 (#102)
OpenBAS-Platform/docs: docs/usage/injects.md

Manual action reminders are injects designed to prompt animation team to perform specific actions manually. It allows to place in the timeline a stimulus to be produced manually, outside the platform (e.g. simulated a call from a journalist on the switchboard telephone). These reminders ensure that critical tasks are completed as part of the simulation, enhancing the accuracy and realism of the exercise.

The inject associated with this type is referred to as `Manual`. To be able to log events not directly related to an email or a sms, you can attach manual expectation to this events (see [Manual Expectations](https://docs.openbas.io/latest/usage/expectations/?h=manual#manual-expectations)). 

> ####Example of manual inject:

Reviewer: #### Heading level 4 with a space 

```suggestion
#### Example of a manual inject:
```

- "A crisis cell has been put together"

> ####Manual expectations:

Reviewer: ```suggestion
#### Manual expectations:
```

> - "The team has answered the call from the journalist"
> - "Analyze the situation and identify the issues."
> - "Predicting future developments"
> - "Decide on the actions to be taken"
> - "Coordinate & communicate with internal and external stakeholders."
> - "Establish and share updates of the ongoing situation"

Reviewer: ```suggestion
-"The team responded to the journalist's inquiry."
- "Analyze the situation and identify the key issues."
- "Anticipate future developments."
- "Determine the necessary actions to take."
- "Coordinate and communicate with both internal and external stakeholders."
- "Provide regular updates on the ongoing situation."
```
